WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -154,26 +157,20 @@ class WPMainActivityViewModel @Inject constructor(
 
         _showQuickStarInBottomSheet.postValue(shouldShowQuickStartFocusPoint)
 
-        if (shouldShowStories(site)) {
+        if (shouldShowStories(site) || hasFullAccessToContent(site)) {
+            // The user has at least two create options available for this site (pages and/or story posts),
+            // so we should show a bottom sheet.
+            // Future creation options added in the future should also be weighed here.
```

Comment:
```suggestion
            // Creation options added in the future should also be weighed here.
```

Reply:
What? I wanted it to be clear I was talking about the _**FUTURE**_

... 🤦 thanks, applied the suggestion 😀